app/components/floating_text_input_label/index.tsx

```diff
@@ -169,6 +91,7 @@ const FloatingTextInput = forwardRef<FloatingTextInputRef, FloatingTextInputProp
     ), [onBlur]);
 
     const onTextInputFocus = useCallback((e: NativeSyntheticEvent<TextInputFocusEventData>) => onExecution(e,
+        // eslint-disable-next-line max-nested-callbacks
```

Comment:
same here, extract the function to avoid max nested callbacks, the reason we have that rule is to help with readability